### DISCUSSION TITLE
Does the seal support multiple graphics cards?

### DISCUSSION THREAD

**Discussion by ChaiWenyu1983** [filecoin-project/lotus#9224] [General Q&A] [answered]
I also have idle graphics cards, N cards and a cards. Can I use them on the same computer.

**Comment by rjan90** [comment, marked as answer]
Hey @ChaiWenyu1983! Yes, its possible to run multiple GPUs on the same server. It will need a bit of tweaking on the Lotus side though, as you would need to run a separate `lotus-worker` for each GPU-card 😄
